gulpfile.js

```diff
@@ -1,11 +1,4 @@
 const gulp = require('gulp')
 const rhtmlBuildUtils = require('rhtmlBuildUtils')
-
-// Temporarily skipping makeDocs because circle CI does not install R
-gulp.task('makeDocs', gulp.series(function (done) {
-  console.log('skipping makeDocs')
-  done()
-}))
-
-const dontRegisterTheseTasks = [ 'makeDocs' ]
+const dontRegisterTheseTasks = []
```

Comment:
R docs are still not created on Circle CI, but at least now they will be built of your local machine